### PR TITLE
Add missing step in the MC template

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: docs/add_mc.md [hunk 1]
```diff
@@ -189,6 +189,27 @@ in the `default` namespace.
     EOF
     ```
 
+2. Create the MC kustomization file poiting the git repo and applied it:
```
Comment (pipo02mix) [inline docs/add_mc.md:192]:
```suggestion
2. Create the MC kustomization file pointing to the git repo and apply it:
```

File: docs/add_mc.md [hunk 2]
```diff
@@ -189,6 +189,27 @@ in the `default` namespace.
     EOF
     ```
 
+2. Create the MC kustomization file pointing to the git repo and apply it:
```
Comment (MarcelMue) [inline docs/add_mc.md:192]:
```suggestion
1. Create the MC kustomization file pointing to the git repo and apply it:
```

**Review by MarcelMue — COMMENTED** [review]
@ljakimczuk Can you double check here?

File: docs/add_mc.md [hunk 3]
```diff
@@ -189,6 +189,27 @@ in the `default` namespace.
     EOF
     ```
 
+1. Create the MC kustomization file pointing to the git repo and apply it:
```
Comment (ljakimczuk) [inline docs/add_mc.md:192]:
```suggestion
1. Apply the cluster's Kustomization CR:

```sh
kubectl apply -f management-clusters/${MC_NAME}/${MC_NAME}.yaml
```

**Review by ljakimczuk — CHANGES_REQUESTED** [review]

File: docs/add_mc.md (hunk 3 above)
Comment (ljakimczuk) [reply docs/add_mc.md:192]: we do not need to create this Kustomization CR since it leaves in the repository now, we just need to apply it

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**